composer.json

```
@@ -10,13 +10,13 @@
         }
     ],
     "require" : {
-        "php": "^7.1",
+        "php": "^8.0",
```

Could you please change that to support both 7 & 8? E.g.:
```suggestion
        "php": "^7.3 || ^8",
```

We still use PHP 7.3 in some places and many people have not yet upgraded. If there is no code restriction to support both, I'd prefer support 7.x for some more time.